neural_compressor/torch/algorithms/layer_wise/utils.py

```diff
@@ -241,8 +264,14 @@ def load_value(model, param_name, path):
             if module == input_embeddings:
                 param_name = name + "." + param_name.split(".")[-1]
     prefix = model.base_model_prefix
-    if "pytorch_model.bin.index.json" in os.listdir(path):
-        value = load_tensor_from_shard(path, param_name, prefix)
+    files = os.listdir(path)
+    safetensors_files = [filename for filename in files if filename.endswith(".safetensors")]
+    if len(safetensors_files) == 1:
+        value = load_tensor_from_safetensors(os.path.join(path, "model.safetensors"), param_name, device=device)
+    elif len(safetensors_files) >= 2:
+        value = load_tensor_from_safetensors_shard(path, param_name, device=device)
+    elif "pytorch_model.bin.index.json" in files:
+        value = load_tensor_from_shard(os.path.join(path, "model.safetensors"), param_name, prefix)
```

Comment:
mistake? `pytorch_model.bin.index.json` has no relationship with `model.safetensors`

Reply:
thanks, fixed in https://github.com/intel/neural-compressor/pull/2039/commits/2129543edd5196a75d9f5f6c0915ec456ab9792d